src/cbc_sdk/enterprise_edr/base.py

```diff
@@ -612,3 +627,179 @@ def _perform_query(self):
             results["incomplete_results"] = result["incomplete_results"]
 
         return results
+
+
+class AsyncEventQuery(Query):
```

Comment:
This should probably be called AsyncEnrichedEventQuery

Reply:
Done.

Reply:
Done.